### ISSUE TITLE
Insufficient color contrast for table captions

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-book#1004]
UT-Arlington reports that table caption color which isn't sufficient for visibility on the standard white background. Can fix by making `#888` a darker color that complies with 4.5:1 ratio recommended by WCAG (like `#666` for example). 

![Screenshot from 2022-04-08 09-12-43](https://user-images.githubusercontent.com/13485451/162481454-ff079c56-7589-4a66-b814-1a377c71e6bf.png)

**Comment by SteelWagstaff** [comment]
Tested and working as expected on integrations. Ready for release.
